readthedocs/projects/migrations/0024_add-show-version-warning.py

```diff
@@ -15,7 +15,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='project',
             name='show_version_warning',
-            field=models.BooleanField(default=False, help_text='Show warning banner in non-stable nor latest versions.', verbose_name='Show version warning'),
+            field=models.URLField(blank=True, help_text='You are seeing unstable version of the documentation, see the stable versions.', verbose_name='Show version warning'),
```

Comment:
There is not need to change the field type. Also, the text doesn't match the usage of this field.

Reply:
I don't know what you are saying. How do you think you will have a link shown in the banner?